apps/portal/src/app/contracts/faqs/page.mdx

```diff
@@ -0,0 +1,35 @@
+import { Details } from "@doc";
+
+export const metadata = {
```

Comment:
use `createMetadata` function for creating metadata. Look at the pages for reference